pkg/cmd/ingest.go

```diff
@@ -54,31 +56,46 @@ This is a constraint of the Vector Database and Similarity Search, as different
 }
 
 func (s *ClientIngest) Run(cmd *cobra.Command, args []string) error {
-	c, err := s.getClient(cmd.Context())
+	filePath := args[0]
+	err := s.run(cmd.Context(), filePath)
 	if err != nil {
-		return err
+		exitErr0(err)
```

Comment:
Do you have to do this because otherwise cobra will return a non-zero if you bubble an err all the way up?

Reply:
Yup